spec/controllers/attachments_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+describe AttachmentsController, type: :controller do
+  let(:board) {FactoryBot.create :board}
+  let(:list) {FactoryBot.create :list, board: board}
+  let(:tag) {FactoryBot.create :tag, list: list}
+  let(:user) {FactoryBot.create :user}
+  let(:user_board) {FactoryBot.create :user_board, user: user, board: board, role_id: 1}
+
+  before do
+    login user
+    set_permission user, board
+  end
+
+  describe "actions" do
+    describe "POST #create" do
+      let(:params) {{attachment: {content: Rack::Test::UploadedFile.new("#{Rails.root}/spec/fixtures/files/avatar.jpg")}, board_id: board.id, tag_id: tag.id}}
+
+      context "when create attachment success" do
+        before {post :create, params: params, xhr: true}
+        it "is expected to add a new attachment" do
+          expect(Attachment.count).to eq 1
```

Comment:
đều expect như nhau
expect(Attachment.count)
thì e dùng subject đi
để ở ngoài context
subject{Attachment.count}
thay toàn bộ expect(Attachment.count).to eq thành is_expected.to eq

Reply:
vâng ạ